profiler/src/ProfilerEngine/Datadog.Profiler.Native/CrashReporting.cpp

```diff
@@ -311,4 +312,20 @@ int32_t CrashReporting::WriteToFile(const char* url)
     }
 
     return 0;
+}
+
+int32_t CrashReporting::CrashProcess()
```

Comment:
I mean, it's obvious... but is there an `[obsolete]` attribute or anything we could throw on too?

```suggestion
// Used for testing, obviously don't ever call this directly!
int32_t CrashReporting::CrashProcess()
```

Reply:
I don't think there is an obsolete attribute in C++, but I added it to the C# interface

Reply:
Not the same name `deprecated`
https://en.cppreference.com/w/cpp/language/attributes/deprecated

(unresolving the conversation so you see my comment. Feel free to ignore it since you've done it in the C# interface)